gdal/ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobuflayer.cpp

```diff
@@ -223,8 +223,22 @@ const std::vector<Offset<Column>> OGRFlatGeobufLayer::writeColumns(FlatBufferBui
         const auto field = m_poFeatureDefn->GetFieldDefn(i);
         const auto name = field->GetNameRef();
         const auto columnType = toColumnType(field->GetType(), field->GetSubType());
+        const auto title = field->GetAlternativeNameRef();
```

Comment:
GetAlternativeNameRef() never returns a nullptr, but an empty string if it is not set. I guess you should test for that, and if that's empty set the flatgeobuf field to nullptr